### DISCUSSION TITLE
JMS Sender with message properties

### DISCUSSION THREAD

**Discussion by samoanGuy7** [nextgenhealthcare/connect#4570] [Show and tell]
Here's some code I have that creates a jms sender and allows properties on the message object, not yet natively supported by mirth connect.

https://github.com/samoanGuy7/mirth-connect-JMS-sender-with-headers

**Comment by twest-mirthconnect** [comment]
@cturczynskyj  @pladesma @narupley - Thoughts on this for the community collaboration this next sprint?

**Reply by cturczynskyj** [reply]
> @cturczynskyj @pladesma @narupley - Thoughts on this for the community collaboration this next sprint?

It looks like there's an [issue created for this](https://github.com/nextgenhealthcare/connect/issues/3486) and that @samoanGuy7 just wanted to discuss their workaround.
